docs/contributing/hacking.rst

```diff
@@ -139,10 +313,38 @@ Make sure it meets the following requirements:
     ``172.16.0.0/12``, or ``192.168.0.0/16``, you'll have to modify ``contrib/coreos/user-data.example``
     and whitelist your development registry so the daemons can pull your custom components.
 
+Initial Platform Build
+----------------------
+
+The full environment is prepared. You can now build Deis from source code and
+then run the platform.
+
+We'll do three steps together:
+
+- Build the source
+- Update our local cluster with a dev release
+- Start the platform
+
+.. code-block:: console
+
+    $ make build    # Build the entire codebase. This will take a while.
+    $ make dev-release  # Create a local dev release.
+    $ deisctl start platform
```

Comment:
Interesting. The order in which they shut down is different depending on whether you do `make restart` or `deisctl platform stop`.